Symfony/CS/DocBlock/Tag.php

```diff
@@ -46,10 +46,9 @@ public function __construct($content)
     {
         preg_match_all('/@[a-zA-Z0-9_]+/', $content, $matches);
 
+        $this->name = 'other';
         if (isset($matches[0][0])) {
             $this->name = strtolower(ltrim($matches[0][0], '@'));
-        } else {
-            $this->name = 'other';
```

Comment:
Don;t like that change. could you elaborate on it?
If first condition is matched then, id "old" code, there will be only one assignment to `$this->name`, now it would be two.